tests/webdriver-e2e/page-objects/common.page.js

```diff
@@ -0,0 +1,18 @@
+class CommonPage {
+  get hamburgerMenu() { return $('#header-dropdown-link'); }
+  get logoutButton() { return $('.fa-power-off'); }
+  get modalBody() { return $('div.modal-body'); }
+  get yesButton() { return $('a.btn.submit.btn-danger'); }
+  // const modal = $('div.modal-dialog');
+  async logout() {
+    await (await this.hamburgerMenu).click();
+    await (await this.logoutButton).click();
+    (await this.modalBody).waitForDisplayed();
+    const warning = await (await this.modalBody).getText();
+    await (await this.yesButton).click();
+    // await helper.waitUntilReadyNative(element(by.css('form#form')));
+    return warning;
+  }
+}
+
+module.exports = new CommonPage();
```

Comment:
I'm interested in this design choice. 
Why did you choose to use a class if you're not planning to use it as a class? You immediately instantiate it here, and return an instance. All this code could have just been inline constants and functions, just like our existent page-objects files. 

We're not class based anywhere else. Why start here without having a specific need for classes in the first place?

Reply:
And I just saw that your `page-objetcs/login.page.js` is procedural :) 
So, I guess, I vote for consistency and refactoring this to not be class based.

Reply:
Aaand I just read your comment in the PR description. 
Maybe this will help: https://stackoverflow.com/a/56065741
Docs say to use `browser.$`: https://webdriver.io/docs/api/browser/$/

Reply:
I tried those briefly yesterday but I'll tiimebox this on getting it to work in a structure that matches the rest of our code. I'll reach out if I can't